modules/qna/src/views/full/ImportModal.tsx

```diff
@@ -0,0 +1,236 @@
+import { Button, Callout, Classes, Dialog, FileInput, FormGroup, Intent, Radio, RadioGroup } from '@blueprintjs/core'
+import 'bluebird-global'
+import _ from 'lodash'
+import React, { FC, Fragment, useEffect, useState } from 'react'
+
+import { toastFailure, toastSuccess } from './toaster'
+
+const JSON_STATUS_POLL_INTERVAL = 1000
+const axiosConfig = { headers: { 'Content-Type': 'multipart/form-data' } }
+
+interface Props {
+  axios: any
+  onImportCompleted: () => void
+}
+
+interface Summary {
+  qnaCount: number
+  cmsCount: number
+  fileQnaCount: number
+  fileCmsCount: number
+}
+
+export const ImportModal: FC<Props> = props => {
+  const [file, setFile] = useState<any>()
+  const [filePath, setFilePath] = useState<string>()
+  const [isLoading, setIsLoading] = useState(false)
+  const [isDialogOpen, setDialogOpen] = useState(false)
+  const [importAction, setImportAction] = useState('insert')
+  const [summary, setSummary] = useState<Summary>()
+  const [statusId, setStatusId] = useState<string>()
+  const [uploadStatus, setUploadStatus] = useState<string>()
+  const [hasError, setHasError] = useState(false)
+
+  useEffect(() => {
+    if (statusId) {
+      const interval = setInterval(async () => {
+        await updateUploadStatus()
+      }, JSON_STATUS_POLL_INTERVAL)
+      return () => clearInterval(interval)
+    }
+  }, [statusId])
+
+  const queryImportSummary = async () => {
```

Comment:
I still don't get why one would import a summary ? Isn't the summary computed before import